changelogs/fragments/changelogs/fragments/1418_vmware_host_vmnic_info.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- vmware_host_vmnic_info - add CDP information to output when applicable (https://github.com/ansible-collections/community.vmware/pull/1418).
```

Comment:
Just `changelogs/fragments/1418_vmware_host_vmnic_info.yml`. There are one `changelogs/`and one `fragments/` too many.